test/cli/test-mutating-admission-policy/Check-replica/policy.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: admissionregistration.k8s.io/v1alpha1
+kind: MutatingAdmissionPolicy
+metadata:
+  name: sample-policy
+spec:
+  matchConstraints:
+    resourceRules:
+    - apiGroups:
+      - apps
+      apiVersions:
+      - v1
+      operations:
+      - CREATE
+      - UPDATE
+      resources:
+      - deployments
```

Comment:
Can you rewrite it in the following format?
```
    - apiGroups: [ "apps" ]
      apiVersions: [ "v1" ]
      operations: [ "CREATE" , "UPDATE"]
      resources: [ "deployments" ]
```
It is important to have a consistent way of writing policies.